mods/boats/init.lua

```diff
@@ -181,15 +181,12 @@ function boat.on_step(self, dtime)
 		return
 	end
 	local s = get_sign(self.v)
-	self.v = self.v - dtime * 0.6 * s
+	self.v = self.v - s * dtime * 0.2 * self.v * self.v
```

Comment:
This currently changes the maximum velocity.
```
a = 4.0
v_max = 5
c * v_max * v_max = a
c = a / v_max / v_max = 4/25 = 0.16 ~= 0.2
```
I suggest to use `0.16` instead of `0.2` or/and to change the boats acceleration.

Reply:
Nice catch ! Thanks.

I have increased the boat acceleration from 4.0 to 5.0.

I have also optimized code a little bit. Now it is much faster than previously.
The code could be optimized further, but I would like to leave it for next Pull Request.